tests/test_mint_api.rs

```diff
@@ -160,44 +160,67 @@ async fn test_get_asset_proof_failure() {
     assert!(result.is_err(), "Expected an error but got success");
 }
 
-#[tokio::test]
-async fn test_mint_api_authority_from_cluster_success() {
-    let devnet_cluster: Cluster = Cluster::Devnet;
-    let mainnet_cluster: Cluster = Cluster::MainnetBeta;
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use solana_program::pubkey::Pubkey;
+    use std::str::FromStr;
 
-    let devnet_authority: std::result::Result<MintApiAuthority, &str> = MintApiAuthority::from_cluster(devnet_cluster);
-    let mainnet_authority: std::result::Result<MintApiAuthority, &str> =
-        MintApiAuthority::from_cluster(mainnet_cluster);
+    #[test]
+    fn test_into_pubkey() {
+        let pubkey_str = "HnT5KVAywGgQDhmh6Usk4bxRg4RwKxCK4jmECyaDth5R";
+        let expected_pubkey = Pubkey::from_str(pubkey_str).unwrap();
 
-    assert_eq!(
-        devnet_authority.unwrap(),
-        MintApiAuthority::Devnet("2LbAtCJSaHqTnP9M5QSjvAMXk79RNLusFspFN5Ew67TC"),
-        "Devnet authority did not match expected value"
-    );
-    assert_eq!(
-        mainnet_authority.unwrap(),
-        MintApiAuthority::Mainnet("HnT5KVAywGgQDhmh6Usk4bxRg4RwKxCK4jmECyaDth5R"),
-        "Mainnet authority did not match expected value"
-    );
-}
+        let mint_authority = MintApiAuthority::Mainnet(expected_pubkey);
+        let converted_pubkey: Pubkey = mint_authority.into();
 
-#[tokio::test]
-async fn test_mint_api_authority_from_cluster_failure() {
-    let devnet_cluster: Cluster = Cluster::Devnet;
-    let mainnet_cluster: Cluster = Cluster::MainnetBeta;
-
-    let devnet_authority: std::result::Result<MintApiAuthority, &str> = MintApiAuthority::from_cluster(devnet_cluster);
-    let mainnet_authority: std::result::Result<MintApiAuthority, &str> =
-        MintApiAuthority::from_cluster(mainnet_cluster);
-
-    assert_ne!(
-        devnet_authority.unwrap(),
-        MintApiAuthority::Devnet("Blade"),
```

Comment:
Only fitting that this is merged on the same day the new Blade movie is delayed indefinitely